docs/code/ft.rst

```diff
@@ -0,0 +1,147 @@
+.. role:: hidden
+    :class: hidden-section
+
+ft
+*******
```

Comment:
I am wondering whether we need to document the api for all generated ontology classes.

Reply:
yes, I mean I am considering whether we should include these. Maybe it is a bit much to write all the classes in documentation from the generated classes. But I am not sure of the answer yet.